Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -447,6 +452,7 @@ import BraintreeDataCollector
             if didAppear {
                 apiClient.sendAnalyticsEvent(
                     BTPayPalAnalytics.browserPresentationSucceeded,
+                    isConfigFromCache: isConfigFromCache,
```

Comment:
Do we only care about this for `browserPresentationSucceeded` or are there any other events we may want to send this for?

Reply:
I wanted to get folks feedback on that. Currently this is the only event we are using it for in our dashboard, but do you think there are others it may be useful?

Was operating under not adding it to the others unless there was a clear need - but open to your thoughts

Reply:
In that case, we should just keep it to this event. If we need to add it elsewhere we can at a later date.